test/python/transpiler/test_check_map.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Test the Check Map pass"""
+
+import unittest
+
+from qiskit import QuantumRegister, QuantumCircuit
+from qiskit.transpiler.passes import CheckMap
+from qiskit.mapper import Coupling
+from qiskit.dagcircuit import DAGCircuit
+from ..common import QiskitTestCase
+
+
+class TestCheckMap(QiskitTestCase):
+    """ Tests the CheckMap pass."""
+
+    def test_trivial_map(self):
+        """ Trivial map in a circuit without entanglement
+         qr0:---[H]---
+
+         qr1:---[H]---
+
+         qr2:---[H]---
+
+         Coupling map: None
+        """
+        qr = QuantumRegister(3, 'qr')
+        circuit = QuantumCircuit(qr)
+        circuit.h(qr)
+        coupling = Coupling()
+        dag = DAGCircuit.fromQuantumCircuit(circuit, expand_gates=False)
```

Comment:
can you remove the `expand_gates` arguments in these tests? they're not used here anyway since default is False. but also I would like to deprecate this arg (with the philosophy of all unrolling and gate expansion should be handled by the unroller pass).

Reply:
Fixed in e38bcd8407d3bc9ee6504279fc539ff36a6f9594